src/authentication/authentication-hapi-plugin.ts

```diff
@@ -2,72 +2,38 @@
 import * as Boom from 'boom';
 import * as auth from 'hapi-auth-jwt2';
 import { inject, injectable } from 'inversify';
-import * as Joi from 'joi';
 import * as qs from 'querystring';
 
 import * as Hapi from '../server/hapi';
 import { HapiPlugin } from '../server/hapi-register';
 import { Group, User } from '../shared/gitlab';
 import { GitlabClient } from '../shared/gitlab-client';
 import { Logger, loggerInjectSymbol } from '../shared/logger';
-import {default as AuthenticationModule} from './authentication-module';
-import { AccessToken, authServerBaseUrlInjectSymbol, jwtOptionsInjectSymbol } from './types';
-
-interface UserId {
-  id: string;
-  idp: string;
-}
+import AuthenticationModule from './authentication-module';
+import { AccessToken, jwtOptionsInjectSymbol } from './types';
 
 @injectable()
 class AuthenticationHapiPlugin extends HapiPlugin {
 
   public static injectSymbol = Symbol('authentication-hapi-plugin');
-  private readonly authenticationModule: AuthenticationModule;
-  private readonly authServerBaseUrl: string;
-  private readonly logger: Logger;
-  private readonly gitlab: GitlabClient;
-  private readonly options: auth.JWTStrategyOptions;
-  private teamTokenClaimKey: 'https://team_token' = 'https://team_token';
-  private subEmailClaimKey: 'https://sub_email' = 'https://sub_email';
+  private readonly subEmailClaimKey: 'https://sub_email' = 'https://sub_email';
 
   constructor(
-    @inject(AuthenticationModule.injectSymbol) authenticationModule: AuthenticationModule,
-    @inject(authServerBaseUrlInjectSymbol) authServerBaseUrl: string,
-    @inject(loggerInjectSymbol) logger: Logger,
-    @inject(GitlabClient.injectSymbol) gitlab: GitlabClient,
-    @inject(jwtOptionsInjectSymbol) jwtOptions: auth.JWTStrategyOptions,
-    ) {
+    @inject(AuthenticationModule.injectSymbol) readonly authenticationModule: AuthenticationModule,
```

Comment:
Shouldn't it be `private readonly authenticationModule`, etc.? Members are, by default, public.